docs/content/docs/custom-resource/reference.md

```diff
@@ -24,4 +22,150 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Reference
+# FlinkDeployment Reference
+## Spec
+
+### FlinkDeploymentSpec
+**Class**: org.apache.flink.kubernetes.operator.crd.spec.FlinkDeploymentSpec
+
+**Description**: Spec that describes a Flink application deployment.
```

Comment:
"Flink application" is a little misleading since it could be an application or session cluster.